llvm/lib/Target/AMDGPU/VOP3PInstructions.td

```diff
@@ -360,7 +360,7 @@ class UDot2Pat<Instruction Inst> : GCNPat <
                              (and i32:$src1, (i32 65535)))
    ),
   (Inst (i32 8), $src0, (i32 8), $src1, (i32 8), $src2, (i1 0))> {
-  let SubtargetPredicate = !cast<VOP_Pseudo>(Inst).SubtargetPredicate;
+  let Predicates = !cast<VOP_Pseudo>(Inst).Predicates;
```

Comment:
As a follow up maybe change the template argument to `VOP_Pseudo Inst` to avoid the cast?

Reply:
https://github.com/llvm/llvm-project/pull/71802